core-kit/curated/sys-fs/zfs/autogen.yaml

```diff
@@ -6,13 +6,6 @@ zfs_and_friends:
       user: openzfs
       repo: zfs
       query: releases
-      match: zfs-(2\.2\.[0-9].*)
-#      include:
-#        - prerelease
-#      transform:
-#        - kind: string
-#          match: '-rc'
-#          replace: '_rc'
```

Comment:
We can maintain the transform in order to avoid having broken ebuild with new release candidates and not gentoo like version string.

Reply:
Like described in the other section we keep stable release here.